lib/ruby-stackoverflow/client/badges_helper.rb

```diff
@@ -2,48 +2,48 @@ module RubyStackoverflow
   class Client
     module BadgesHelper
 
-      def badges(options={})
+      def badges(options = {})
         badges_response(options)
-      end 
+      end
 
-      def badges_by_ids(ids, options={})
+      def badges_by_ids(ids, options = {})
         ids = join_ids(ids)
-        url = ids 
+        url = ids
         badges_response(options, url)
-      end 
+      end
 
-      def badges_by_ids(ids, options={})
+      def badges_by_ids(ids, options = {})
         ids = join_ids(ids)
-        url = ids 
+        url = ids
         badges_response(options, url)
-      end 
+      end
 
-      def badges_by_name(options={})
-        url ='/name'
+      def badges_by_name(options = {})
+        url = '/name'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.